src/python/WMCore/FwkJobReport/Report.py

```diff
@@ -579,6 +580,7 @@ def addError(self, stepName, exitCode, errorType, errorDetails):
         errDetails.details = errorDetails
 
         setattr(stepSection.errors, "errorCount", errorCount + 1)
+        self.setStepStatus(stepName=stepName, status=exitCode)
```

Comment:
I wonder how all the other steps and job type work just fine if we were missing this setStepStatus?

Reply:
I seems most cases, error status checked by error count not status, but there are one place it is checked by status.